core/tests/protractor_utils/GetStartedPage.js

```diff
@@ -37,6 +37,7 @@ var GetStartedPage = function() {
       throw new Error('Unsupported tag type specified: ' + type);
     }
 
+    waitFor.visibilityOf(tag, 'Tag is taking too long to appear');
```

Comment:
You do need an `await` here. See my earlier comment

Reply:
Done